engine/glfw/lib/android/android_util.c

```diff
@@ -123,10 +124,39 @@ static EGLint choose_egl_config(EGLDisplay display, EGLConfig* config)
     return result;
 }
 
+static int IsAppAndWindowReady(_GLFWwin_android* win)
+{
+    return win != 0 && win->app != 0 && win->app->window != 0;
```

Comment:
Our working theory is that the window is 0 for some reason, and that there's a race condition between the looper and game threads during `sys.reboot()`
I.e we think the game thread restarts the engine, and tries to open a window before the OS has assigned a new window to the app.

So, let's wait for a bit, to see if that helps!

(We haven't been able to reproduce it locally)